README.md

```diff
@@ -13,7 +13,7 @@ This tool, henceforth referred to simply as "the runtime", builds upon
 the [virtcontainers](https://github.com/containers/virtcontainers)
 project to provide a high-performance standards-compliant runtime that
 creates hardware-virtualized containers which leverage
-[Intel](https://www.intel.com/)'s VT-x technology.
+[Intel](https://www.intel.com/)'s [VT-x technology](https://www.intel.co.uk/content/www/uk/en/virtualization/virtualization-technology/intel-virtualization-technology.html).
```

Comment:
Interesting, I wonder if there's a non-localized, generic link for this. Even intel.com adds us/en.